cpp/bin/torchtrtc/main.cpp

```diff
@@ -432,12 +441,12 @@ int main(int argc, char** argv) {
 
       for (size_t i = 0; i < trt_results.size(); i++) {
         if (!torchtrtc::accuracy::almost_equal(
-                jit_results[i], trt_results[i].reshape_as(jit_results[i]), threshold_val)) {
+                jit_results[i], trt_results[i].reshape_as(jit_results[i]), atol_val, rtol_val)) {
           std::ostringstream threshold_ss;
-          threshold_ss << threshold_val;
+          threshold_ss << "atol: " << atol_val << " rtol: " << rtol_val;
           torchtrt::logging::log(
               torchtrt::logging::Level::kWARNING,
-              std::string("Maximum numerical deviation for output exceeds set threshold (") + threshold_ss.str() +
+              std::string("Maximum numerical deviation for output exceeds tolerance thresholds (") + threshold_ss.str() +
                   std::string(")"));
         }
```

Comment:
Can we add a case where it meets the threshold and prints out what the difference was and the tolerance to LOG_INFO?

Reply:
Shouldn't we dump the threshold and difference only in case of failure?
It will flood the logs with so many test cases.

Reply:
We should log it but users won't see it unless they turn on verbose mode

Reply:
Updated the logging.